ports/soci/portfile.cmake

```
@@ -23,6 +23,10 @@ foreach(_feature IN LISTS ALL_FEATURES)
     else()
         list(APPEND _COMPONENT_FLAGS "-DWITH_${_FEATURE}=OFF")
     endif()
+
+    if ("mysql" IN_LIST FEATURES)
```

```suggestion
    if (_feature MATCHES "mysql")
```

Do not add the options several times.